src/odemis/util/test/transform_test.py

```diff
@@ -601,6 +601,7 @@ def test_rigid_transform_from_pointset_known_values(self):
             tform = RigidTransform.from_pointset(src, dst)
             self.assertAlmostEqual(0., _angle_diff(rotation, tform.rotation))
             numpy.testing.assert_array_almost_equal(translation, tform.translation)
+            self.assertAlmostEqual(tform.fre(src, dst), 0)
```

Comment:
Could you add a test case where .fre() doesn't return 0? 
Maybe it's just a matter of extending src & dst with some non matching values, and checking that FRE increases?

Reply:
I added a test in `SimilarityTransformKnownValues.test_similarity_transform_from_pointset_umeyama`